### ISSUE TITLE
annotation field in profile //alter/add

### ISSUE THREAD

**Issue by brianrufgsa** [usnistgov/OSCAL#499]
# Describe the bug

We added <annotation> as a third option, where prop and part were generally available; however, annotation is not available when using a profile to alter a control.

# Who is the bug affecting?
Anyone who needs to add or change an annotation element in a control using profile syntax.

# What is affected by this bug?

Resolved profiles

# When does this occur?

In profile syntax.

# How do we replicate the issue?

Attempt to use the following in a profile:
```
<alter control-id="ac-1">
	<add position="ending">
		<**annotation** name='core' class='designation' ns='fedramp' />
	</add>
</alter>
```

# Expected behavior (i.e. solution)

The above code should be valid.

# Other Comments

Annotation is a great third option to prop and part. Further, it offers FedRAMP a better way to flag controls in the FedRAMP Tailored baseline (assess, attest, nso, conditional, etc.), and in the High, Moderate, and Low FedRAMP baselines (core), because it is the best construct for attaching notes directly to the label. 
This is especially useful for "conditional" controls in Tailored, where there is a separate condition stating the condition.. 
Having the ability to use

**Comment by brianrufgsa** [comment]
Tagging @david-waltermire-nist and @wendellpiez on this.

**Comment by wendellpiez** [comment]
In PR #516, awaiting review.

**Comment by david-waltermire-nist** [comment]
This was addressed in PR #516.
